app/views/sessions/new_sign_in.html.erb

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
add any block with styling for showing errors when something wrong
it can be or `flash` or instance variable in which you will put errors